docs/hacking/chrome-extension.rst

```diff
@@ -20,7 +20,7 @@ To build and install the Chrome extension:
 
    .. code-block:: bash
 
-      hypothesis-buildext chrome --debug --service 'http://127.0.0.1:5000'
+      hypothesis-buildext chrome --debug --service 'http://127.0.0.1:5000' --bouncer-url "http://127.0.0.1:8000/*"
```

Comment:
Don't bother adding this here. If you want to document it in a subsequent section that explains what the bouncer is and why you might want to build the extension with support for it, that's fine, but not everyone needs to build the extension with support for the bouncer.